### ISSUE TITLE
Use temp-mail.ru in place of guerrilla-mail

### ISSUE THREAD

**Issue by aslafy-z** [aslafy-z/ptc-account-generator#8]
It might be a better option

**Comment by timlinde** [comment]
Sounds good. I looked into that later. Are you gonna implement it?

**Comment by aslafy-z** [comment]
It could be a good option, but there is no wrapper to wait for an email like guerrilla-mail package had.
Are you interested into PRing this ?

**Comment by timlinde** [comment]
I am interested however like another issue said it seems to be blocked by Niantec so we might need to change to another provider if Guerrilla is blocked.
